xamarin/Xamarin.Forms
# Changed the namespace for all the Issues in the control gallery to *Xamarin.Forms.Controls.Issues*

### Description of Change ###

Changed namespaces of all UI tests different of **Xamarin.Forms.Controls.Issues**.

During the refactoring of the namespaces some problems was found in compilation. It was necessary move some helper classes that were inside UI tests to another structure using **Xamarin.Forms.Controls** namespace. I created a new folder called **Controls** with all this refactoring.

### Issues Resolved ### 

- fixes #4150

### API Changes ###

None

### Platforms Affected ### 

All platforms that contains UI tests.

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 

Not applicable

### Testing Procedure ###

Run UI tests on all platforms to check if all works fine. 
In my compilation test, I tested Android, GTK, iOS, MacOS and Tizen compilations of the ControlGallery project. WPF and WindowsUniversal platforms I could not perform the compilation test.

### PR Checklist ###
